### PR TITLE
fix redis doc reference to "opentelemetry-instrumentation" executable

### PR DESCRIPTION
**Pull request by pmcollins**
# Description

The documentation rendered [here](https://opentelemetry-python-contrib.readthedocs.io/en/latest/instrumentation/redis/redis.html) refers to the `opentelemetry-instrumentation` executable which, to my knowledge, should be `opentelemetry-instrument`.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

This is just a docs fix.

# Does This PR Require a Core Repo Change?

- [ ] Yes. - Link to PR: 
- [x] No.

# Checklist:

See [contributing.md](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/CONTRIBUTING.md) for styleguide, changelog guidelines, and more.

- [ ] Followed the style guidelines of this project
- [x] Changelogs have been updated
- [ ] Unit tests have been added
- [x] Documentation has been updated

### CODE REVIEW

**Comment by jeremydvoss** [comment]
lgtm but ther's a cancelled check blocking.

**Comment by pmcollins** [comment]
Looks like the test failure was because httpbin was unavailable during the test run. Have folks considered replacing httpbin with a local test server -- if so perhaps creating a new issue is in order? cc: @ocelotl

**Comment by shalevr** [comment]
> Looks like the test failure was because httpbin was unavailable during the test run. Have folks considered replacing httpbin with a local test server -- if so perhaps creating a new issue is in order? cc: @ocelotl

There is an open issue for this #1681. I will fix this soon. In the meanwhile, we will re-run this test

**Comment by shalevr** [comment]
And for the cancelled check, it will be fixed after #1826 will merge

**Review by shalevr — APPROVED** [review]
Thank you

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+- `opentelemetry-instrumentation-redis` Fix documentation reference to nonexistent `opentelemetry-instrumentation` command
```
Comment (shalevr) [inline CHANGELOG.md:10]: The changelog entry is not necessary for docs update
```suggestion
```

**Pull request merged**